team-norms/timesheets.md

```diff
@@ -0,0 +1,32 @@
+# Timesheets
+
+If you work on a Delivery Team, you are expected to complete a timesheet to show where your time has been spent during each week.
+
+Accurate timesheeting is really important to the business, as we generate customer invoices direct from timesheets. Not completing timesheets means we give your hard work away for free.
+
+The good news is that at Made Tech, Engineers are dedicated to one engagement at any one time, and Delivery Managers are typically likely to be split across a couple of engagements at most. For most team members completing a weekly timesheet should take less than 30 seconds.
+
+## General guidance
+
+- Every billable team member must completed a timesheet attributing a 35 hour week (pro-rated for those who do not work a full 5 day week) by 1.30pm every Friday
+
+- If the last day of the month falls on a working day, on the day, you will be asked to complete your timesheets up to and including the final day of the month. You must do this by 1.30pm
+
+- Timesheets should typically be completed in 3.5 hour increments. 3.5 hours for a half day, 7 hours for a full day
+
+- If you take an hour or so out of your day for an appointment, or work an extra hour or so in to the evening, we do not typically reflect this on timesheets and so you should bill your regular 7 hour day
+
+
+## Categories
+
+- *Project*: by default you're likely to be focused on a single engagement at any one time, and so you should timesheet 7 hours per working day against this engagement
+
+- *Learn Tech*: if you partake in Learn Tech you should timesheet 3.5 hours on a Friday afternoon against Learn Tech, and 3.5 hours against your current engagement
+
+- *Holidays*: prior to heading off on holiday, as well as completing any outstanding timesheets, and putting your out of office autoresponder on, you must proactively complete timesheets for the duration of your holiday, booking your time against Made Tech -> Holiday
```

Comment:
Would suggest we call this absence, as this covers medical (for appointments), compassionate, etc.